Panic with 1.0.0 when using unspecified method and multiple params

This started with 1.0.0.

Error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x69823b]

goroutine 20 [running]:
github.com/valyala/fasthttp.(*RequestCtx).SetUserValue(...)
	/home/greg/go/pkg/mod/github.com/valyala/fasthttp@v1.9.0/server.go:581
github.com/fasthttp/router/radix.(*node).getFromChild(0xc0000b45a0, 0xc00001e1d6, 0xa, 0x0, 0x10, 0x9592bc)
	/home/greg/go/pkg/mod/github.com/fasthttp/router@v1.0.0/radix/node.go:307 +0x59b
github.com/fasthttp/router/radix.(*node).getFromChild(0xc0000b4420, 0xc00001e1d1, 0xf, 0x0, 0x410404, 0xc000057a10)
	/home/greg/go/pkg/mod/github.com/fasthttp/router@v1.0.0/radix/node.go:302 +0x4b1
github.com/fasthttp/router/radix.(*Tree).Get(0xc0000b2040, 0xc00001e1d0, 0x10, 0x0, 0xc00009d678, 0x10)
	/home/greg/go/pkg/mod/github.com/fasthttp/router@v1.0.0/radix/tree.go:88 +0x1f7
github.com/fasthttp/router.(*Router).allowed(0xc0000b4360, 0xc00001e1d0, 0x10, 0xc00001e138, 0x4, 0x0, 0x3200000000000000)
	/home/greg/go/pkg/mod/github.com/fasthttp/router@v1.0.0/router.go:405 +0x5dc
github.com/fasthttp/router.(*Router).Handler(0xc0000b4360, 0xc000078000)
	/home/greg/go/pkg/mod/github.com/fasthttp/router@v1.0.0/router.go:511 +0x37a
github.com/valyala/fasthttp.(*Server).serveConn(0xc0000fc000, 0x7a4f40, 0xc0000b2088, 0x0, 0x0)
	/home/greg/go/pkg/mod/github.com/valyala/fasthttp@v1.9.0/server.go:2041 +0x554
github.com/valyala/fasthttp.(*workerPool).workerFunc(0xc0000beb40, 0xc0000a6460)
	/home/greg/go/pkg/mod/github.com/valyala/fasthttp@v1.9.0/workerpool.go:223 +0xc0
github.com/valyala/fasthttp.(*workerPool).getCh.func1(0xc0000beb40, 0xc0000a6460, 0x6bb4a0, 0xc0000a6460)
	/home/greg/go/pkg/mod/github.com/valyala/fasthttp@v1.9.0/workerpool.go:195 +0x35
created by github.com/valyala/fasthttp.(*workerPool).getCh
	/home/greg/go/pkg/mod/github.com/valyala/fasthttp@v1.9.0/workerpool.go:194 +0x101
```

Code:

```
package main

import (
	"log"

	"github.com/fasthttp/router"
	"github.com/valyala/fasthttp"
)

func main() {
	r := router.New()
	r.GET("/{version}/{service}/{value}", func(ctx *fasthttp.RequestCtx) {})

	log.Fatal(fasthttp.ListenAndServe(":8080", r.Handler))
}
```
Request:

```
curl  -d '{}' 'localhost:8080/does/not/matter'
```

--- a/radix/node.go
+++ b/radix/node.go
@@ -303,8 +303,10 @@ walk:
 				if tsr {
 					return nil, tsr
 				} else if h != nil {
-					for i, key := range child.paramKeys {
-						ctx.SetUserValue(key, values[i])
+					if ctx != nil {
+						for i, key := range child.paramKeys {
+							ctx.SetUserValue(key, values[i])
+						}
 					}
 
 					return h, false

--- a/radix/tree_test.go
+++ b/radix/tree_test.go
@@ -20,28 +20,31 @@ func generateHandler() fasthttp.RequestHandler {
 func testHandlerAndParams(
 	t *testing.T, tree *Tree, requestedPath string, handler fasthttp.RequestHandler, wantTSR bool, params map[string]interface{},
 ) {
-	ctx := new(fasthttp.RequestCtx)
+	for _, ctx := range []*fasthttp.RequestCtx{new(fasthttp.RequestCtx), nil} {
 
-	h, tsr := tree.Get(requestedPath, ctx)
-	if reflect.ValueOf(handler).Pointer() != reflect.ValueOf(h).Pointer() {
-		t.Errorf("Path '%s' handler == %p, want %p", requestedPath, h, handler)
-	}
+		h, tsr := tree.Get(requestedPath, ctx)
+		if reflect.ValueOf(handler).Pointer() != reflect.ValueOf(h).Pointer() {
+			t.Errorf("Path '%s' handler == %p, want %p", requestedPath, h, handler)
+		}
 
-	if wantTSR != tsr {
-		t.Errorf("Path '%s' tsr == %v, want %v", requestedPath, tsr, wantTSR)
-	}
+		if wantTSR != tsr {
+			t.Errorf("Path '%s' tsr == %v, want %v", requestedPath, tsr, wantTSR)
+		}
 
-	resultParams := make(map[string]interface{})
-	if params == nil {
-		params = make(map[string]interface{})
-	}
+		if ctx != nil {
+			resultParams := make(map[string]interface{})
+			if params == nil {
+				params = make(map[string]interface{})
+			}
 
-	ctx.VisitUserValues(func(key []byte, value interface{}) {
-		resultParams[string(key)] = value
-	})
+			ctx.VisitUserValues(func(key []byte, value interface{}) {
+				resultParams[string(key)] = value
+			})
 
-	if !reflect.DeepEqual(resultParams, params) {
-		t.Errorf("User values == %v, want %v", resultParams, params)
+			if !reflect.DeepEqual(resultParams, params) {
+				t.Errorf("User values == %v, want %v", resultParams, params)
+			}
+		}
 	}
 }
 

